src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -129,26 +137,30 @@ foam.CLASS({
       code: function() {
         if ( ! this.stack ) return;
 
-        if ( this.config.createController.class === 'foam.comics.v2.DAOCreateView'){
-          this.stack.push({
-            class: this.config.createController.class,
-            data: ((this.config.factory && this.config.factory$cls) ||  this.data.of).create({ mode: 'create'}, this),
-            config$: this.config$,
-            of: this.data.of
-          }, this.__subContext__);
-        } else if (this.config.createControllerView) {
-          this.stack.push(this.config.createControllerView, this.__subContext__);
+        if ( this.config.createController.class === 'foam.comics.v2.DAOCreateView' ) {
+          this.stack.push(this.StackBlock.create({
+            view: {
+              class: this.config.createController.class,
+              data: ((this.config.factory && this.config.factory$cls) ||  this.data.of).create({ mode: 'create'}, this),
+              config$: this.config$,
+              of: this.data.of
+            }, parent: this.__subContext__
+          }));
+        } else if ( this.config.createControllerView ) {
+          this.stack.push(this.StackBlock.create({ view: this.config.createControllerView, parent: this.__subContext__ }));
         } else {
-          this.stack.push({
-            class: this.config.createController.class,
-            config$: this.config$,
-            of: this.data.of,
-            data: this.selection,
-            detailView: this.config.detailView.class,
-            menu: this.config.menu,
-            controllerMode: foam.u2.ControllerMode.CREATE,
-            isEdit: true
-          }, this.__subContext__);
+          this.stack.push(this.StackBlock.create({
+            view: {
+              class: this.config.createController.class,
+              config$: this.config$,
+              of: this.data.of,
+              data: this.selection,
+              detailView: this.config.detailView.class,
+              menu: this.config.menu,
+              controllerMode: foam.u2.ControllerMode.CREATE,
+              isEdit: true
+            }, parent: this.__subContext__
```

Comment:
Doesn't it work if you just do 'this' instead of this.__subContext__?

Reply:
Not sure why we have these. It should be fine with just `this`. I didn't really edit this part other than modifying the function call. Will update these and see if it breaks things

Reply:
Please test.